### PR TITLE
TELCODOCS-309: BZ1995631 & BZ1999603 / 9/13 Fixed BZs also added to the 4.9 Release Notes

### PR DESCRIPTION
**Pull request by ogradyp**
Applies only to the Enterprise-4.9 branch

https://issues.redhat.com/browse/TELCODOCS-309

Browse the preview for Known Issues: https://deploy-preview-36228--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes#ocp-4-9-known-issues

Browse the preview for Fixed Issues: https://deploy-preview-36228--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes#ocp-4-9-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -1153,6 +1153,9 @@ This script removes unauthenticated subjects from the following cluster role bin
...
 
+* Using the VRF command `ip vrf exec <command>` does not work due to a cgroups mismatch. As a result, this command cannot be used inside OpenShift pods. To use virtual routing and forwarding (VRF), applications must be VRF-aware and bind directly to the VRF interface. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1995631[*BZ#1995631*])
+
+* A non-uniform memory access (NUMA) bug can cause undesired NUMA pinning for the container, which can lead to latency or performance degradation. The Topology Manager can pin the container, under the guaranteed Quality of Service (QoS) pod, with resources that the 'single-numa-node' topology management policy can satisfy to more than one NUMA node. As a workaround, do not start guaranteed QoS pods with container’s memory-resource requests bigger than the 'single-numa-node' policy can provide. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999603[*BZ#1999603*])
```
Comment (ctauchen) [inline release_notes/ocp-4-9-release-notes.adoc:1158]: s/non-uniform/nonuniform
* https://www.merriam-webster.com/dictionary/nonuniform
* see IBMSG 92

The second sentence has a lot of moving parts, and might benefit from being simplified or broken into multiple sentences. After reading through a few times, I can't quite follow what's going on.

**Review by ctauchen — COMMENTED** [review]
Thanks for this -- a few comments and suggestions for you below.

But first, some housekeeping:

Please remember to add a link to your preview content in your first PR comment. This helps the peer review squad find your builds quickly.

**Comment by ctauchen** [comment]
@bergerhoffer @bobfuru FYI

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -1153,6 +1153,9 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * Due to a race condition, the {rh-openstack-first} cloud provider might not start properly. Consequently, LoadBalancer services might never get an `EXTERNAL-IP` set. As a temporary workaround, you can restart the kube-controller-manager pods using the procedure described in link:https://bugzilla.redhat.com/show_bug.cgi?id=2004542[*BZ#2004542*].
 
+* Using the VRF command `ip vrf exec <command>` does not work due to a cgroups mismatch. As a result, this command cannot be used inside OpenShift pods. To use virtual routing and forwarding (VRF), applications must be VRF-aware and bind directly to the VRF interface. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1995631[*BZ#1995631*])
```
[the remark this answers is not in the document]
Comment (bobfuru) [reply release_notes/ocp-4-9-release-notes.adoc:1156]: Agree with @ctauchen here, and wondering what that `<command>` value means. IIUC, the command is actually `ip vrf exec`, and then you specify the pod where you want to run the command in. So I wonder if something simple might make more sense:

> The `ip vrf exec` command does not work due to a cgroups mismatch. 

Then the next two sentences would make more sense to me.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (bobfuru) [reply release_notes/ocp-4-9-release-notes.adoc:1158]: I also find the "workaround" sentence a bit hard to follow. Is it correct to say something like this?

As a workaround, do not start guaranteed QoS pods when the container memory-resource requests are bigger than the 'single-numa-node' policy can provide.

**Review by bobfuru — COMMENTED** [review]
Added a couple of comments for consideration. Thanks!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (ogradyp) [reply release_notes/ocp-4-9-release-notes.adoc:1156]: Thanks for your feedback @ctauchen & @bobfuru . 
My first PR to the openshift-docs repo as I've been working on a standalone conversion to date.
Let me take your feedback to the SMEs to investigate further.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (ogradyp) [reply release_notes/ocp-4-9-release-notes.adoc]: @ctauchen : Thanks for your comment. I have applied 'nonuniform' and split the second sentence in two. 
@bobfuru : Thanks for your comment. I have reworked the 'workaround' sentence as you suggest but just realised it's not in today's version of the .adoc. I have it included now and it will be in the next version of the .adoc in the next days as I have Fixed Issues to include in the Release Notes.

**Review by jeana-redhat — COMMENTED** [review]
In general, there are a handful of terms that I am unsure about. I would expect things like "baremetal-ipi" to be written out (bare metal on installer-provisioned infrastructure?), and things like "ironic-python-agent" to either be enclosed in backticks or written out. But I'm not familiar enough with the bare metal content to know for sure what to recommend - can you check the following against how they are used in the actual content where they are discussed?

- metal3 (I see both metal3 and Metal3 in the repo)
- InitContainers/InitContainer
- baremetal-ipi
- ironic-python-agent
- sushy library
- VirtualMedia.InsertMedia request body
- BareMetalHost
- cgroups

**Review by aireilly — COMMENTED** [review]
A few small comments :)

**Review by ctauchen — COMMENTED** [review]
A few nits, as we're fond of saying.

**Review by ctauchen — COMMENTED** [review]
Just need to clean up the merge conflict helper text. @ogradyp
